extensions/wikia/PaidAssetDrop/PaidAssetDropHooks.class.php

```diff
@@ -0,0 +1,39 @@
+<?php
+class PaidAssetDropHooks {
+
+	/**
+	 * Modify assets appended to the bottom of the page
+	 *
+	 * @param array $jsAssets
+	 *
+	 * @return bool
+	 */
+	public static function onOasisSkinAssetGroups( &$jsAssets ) {
+		global $wgPADConfig;
```

Comment:
Let's use the full name for the wg variable, so we don't wonder in 14 months what the heck is PAD.